projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx

```diff
@@ -135,6 +137,22 @@ const FeaturePromptComponent = props => {
 				strong: <strong />,
 				ExternalLink: <ExternalLink href={ descriptionLink } onClick={ onExternalLinkClick } />,
 			} ) }
+			content={
+				descriptionList || descriptionSecondary ? (
+					<React.Fragment>
+						{ descriptionList && (
+							<ul className="jp-recommendations-question__description-list">
+								{ descriptionList.map( item => (
+									<li>{ item }</li>
```

Comment:
This throws an error in the console because you're missing a `key` attribute

Reply:
I know this was most likely copied from the `resources-prompt`, FWIW in the [Protect PR](https://github.com/Automattic/jetpack/pull/25391) I have a fix for that, so don't feel the need to go fix that there as well (but you totally can if you want haha) but you may want to replicate what I did for that section for consistency (or recommend a different way when we figure out the protect stuff and are reviewing that one)

Reply:
Ah, good catch, thanks! I see that this is fixed for the list in the `ProductSpotlight` component in #25391, I think it should not cause conflicts to fix this for `ResourcePrompt` and `FeaturePrompt` here 👍

Reply:
Well maybe I should have doublechecked before making that comment 😆